examples/step-89/doc/kind

```diff
@@ -0,0 +1 @@
+techniques
```

Comment:
Do we eventually want to have a non-matching "kind"?

Reply:
I think I don't have the background to comment on this. Maybe someone else can comment on this, e.g. @peterrum @kronbichler?